src/diem/testing/miniwallet/app/app.py

```diff
@@ -174,12 +174,12 @@ def _send_external_payment_txn(self, txn: Transaction) -> None:
                 diem_txn = self.diem_account.client.wait_for_transaction(str(txn.signed_transaction), timeout_secs=0.1)
                 self.store.update(txn, status=Transaction.Status.completed, diem_transaction_version=diem_txn.version)
             except jsonrpc.WaitForTransactionTimeout as e:
-                self.store.create_event(txn.account_id, "info", str(e))
+                self.logger.debug("wait for txn(%s) timeout: %s", txn, e)
```

Comment:
given these will be called from different places potentially, why not have both?

Reply:
I think these log / events may need adjust basing on real use cases.
At beginning, I added too much, now remove some of them. Later add some back when found them useful.